rocketc/rocketc.py

```diff
@@ -362,3 +363,12 @@ def _add_to_default_group(self, group_name, user_id):
             self._add_to_group(user_id, group_info['group']['_id'])
             return True
         return False
+
+    def _private_channel(self, room_name):
+        url_search = "{}?{}={}".format("channels.info", "roomName", room_name)
+        channel = self._request_rocket_chat("get", url_search)
```

Comment:
Is it possible to check if the channel is already private? in that way it would not be necessary the second request

Reply:
ping @andrey-canon